nemo/utils/data_utils.py

```diff
@@ -16,15 +16,25 @@
 import pathlib
 import shutil
 import subprocess
-from typing import Tuple
+from functools import lru_cache
+from typing import Any, Callable, Dict, Iterable, Tuple
 from urllib.parse import urlparse
 
 try:
     from nemo import __version__ as NEMO_VERSION
 except ImportError:
     NEMO_VERSION = 'git'
+
 from nemo import constants
 from nemo.utils import logging
+from nemo.utils.nemo_logging import LogMode
+
+try:
+    from lhotse.serialization import open_best as lhotse_open_best
```

Comment:
@ko3n1g could you take a look to confirm this import guard is sufficient to avoid the previous issues in the reverted PR https://github.com/NVIDIA/NeMo/pull/12905 ? Thanks!